hci/event/events/vendor_specific/gap_device_discovery_done.py

```diff
@@ -36,7 +36,7 @@ def __str__(self):
                 GAP_DeviceDiscoveryDone.EventType(self.event_type).name,
                 hex(self.address_type),
                 GAP_DeviceDiscoveryDone.AdressType(self.address_type).name,
-                _bytes_to_hex_string(self.address))
+                _bytes_to_hex_string(self.address, revert=True))
```

Comment:
Properties are responsible for reverting little endian order, not the `__str__` method.